frontend/src/App.tsx

```diff
@@ -72,7 +73,8 @@ const App = (): ReactElement => {
             <Route exact path="/policies" component={Policies} />
             <Route path="/landlord/:landlordId" component={LandlordPage} />
             <Route path="/apartment/:aptId" component={ApartmentPage} />
-            <Route component={NotFoundPage} />
+            <Route exact path="/notfound" component={NotFoundPage} />
+            <Route exact path="/searchresults" component={SearchResultsPage} />
```

Comment:
Also, how about changing this to 
```  <Route path="/search" element={<SearchResultsPage />} />```? That way our search route is more similar to what people do in industry!